pipeline/plugins/operators/custom_operators.py

```diff
@@ -343,27 +344,20 @@ def ssh_execute_and_raise(self, ssh_client, command):
             )
 
     def execute(self, context):
-        tmp_target_path = self.target_path.with_suffix(".zip.tmp")
-
         ssh_hook = SSHHook(ssh_conn_id=self.ssh_conn_id)
         with ssh_hook.get_conn() as ssh_client:
             with ssh_client.open_sftp() as sftp_client:
-                zip_creation_cmd = f"python3 {self.extra_bin_dir/'update_zip_with_sources.py'} {tmp_target_path} --dir {self.data_source}"
-                self.log.info("Creating temporary zip in %s on Puhti", tmp_target_path)
+                zip_creation_cmd = f'/bin/bash -lc ". /appl/profile/zz-csc-env.sh && module load libzip && zipmerge -k {self.target_path} {self.data_source/"*"}"'
```

Comment:
Martin comments: this is so machine-specific that the environment parts could be parametrized.

Reply:
If we already had some parametrization/configuration/whatnot in place, then yes. But we don't, and this doesn't seem to be the most urgent one. I'd say let's reconsider parametrizing this when we have parametrized the more urgent stuff (e.g. hard-coded paths that would cause dev instances to overwrite the prod data).